### PR TITLE
Support specify write type for StressWorkerBench

### PR DESCRIPTION
**Pull request by maobaolong**
Set write type when creating files in stress worker bench

### CODE REVIEW

File: stress/shell/src/main/java/alluxio/stress/cli/worker/StressWorkerBench.java [hunk 1]
```diff
@@ -132,7 +132,7 @@ public void prepare() throws Exception {
       Configuration hdfsConf = new Configuration();
       // force delete, create dirs through to UFS
       hdfsConf.set(PropertyKey.Name.USER_FILE_DELETE_UNCHECKED, "true");
-      hdfsConf.set(PropertyKey.Name.USER_FILE_WRITE_TYPE_DEFAULT, "CACHE_THROUGH");
+      hdfsConf.set(PropertyKey.Name.USER_FILE_WRITE_TYPE_DEFAULT, mParameters.mWriteType);
```
Comment (dbw9580) [inline stress/shell/src/main/java/alluxio/stress/cli/worker/StressWorkerBench.java:135]: justification for the potential impact, quoting previous review comments: https://github.com/Alluxio/alluxio/pull/14842#discussion_r816480093

Comment (dbw9580) [reply stress/shell/src/main/java/alluxio/stress/cli/worker/StressWorkerBench.java:135]: Another issue is that if a user creates the files with `MUST_CACHE` write type, and specifies the `--free` option at the same time, the data will be immediately freed, therefore, in the benchmark stage, workers will not be able to read the data back.

Can you add a check to raise an error if the writetype is `MUST_CACHE` and the user wants to free the blocks?

**Review by dbw9580 — CHANGES_REQUESTED** [review]
Thanks for the PR! I have a minor comment.

File: stress/shell/src/main/java/alluxio/stress/cli/worker/StressWorkerBench.java (hunk 1 above)
Comment (maobaolong) [reply stress/shell/src/main/java/alluxio/stress/cli/worker/StressWorkerBench.java:141]: Done

**Review by dbw9580 — APPROVED** [review]
LGTM

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Pull request merged**